.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: LTE integ test containerized agw
+
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+
+jobs:
+  lte-integ-test-containerized:
+    if: github.repository_owner == 'magma' || github.event_name == 'workflow_dispatch'
```

Comment:
`workflow_dispatch` should be added as an execution option for this workflow

Reply:
done